functests/pvc_creation_test.go

```diff
@@ -43,12 +43,13 @@ var _ = Describe("PVC Creation", func() {
 
 		Context("create pvc", func() {
 			It("and verify bound status", func() {
+				deployManager, err := deploymanager.NewDeployManager()
```

Comment:
Don't do this, this is already taken care of earlier: https://github.com/openshift/ocs-operator/pull/646/files#diff-6b1cc57a75b70f1c156ec6ea5d8e4b43R22

Reply:
Done!